app/controllers/host_controller.rb

```diff
@@ -646,20 +647,18 @@ def get_hosts
   end
 
   def get_session_data
+    super
     @title      = _("Hosts")
     @layout     = "host"
     @drift_db   = "Host"
     @lastaction = session[:host_lastaction]
```

Comment:
`@lastaction` is already handled by GenericSessionMixin .. and so is `@catinfo`.